js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -744,16 +744,35 @@ const DropDownList = DropDownEditor.inherit({
         delete this._searchTimer;
     },
 
-    _updatePopupMinWidth() {
-        windowUtils.hasWindow() && this._popup && this._setPopupOption('minWidth', this.$element().outerWidth());
+    _updatePopupMinWidth(popupWidth) {
+        if(windowUtils.hasWindow() && this._popup) {
```

Comment:
```suggestion
        if(window && this._popup) {
```
[L22](https://github.com/DevExpress/DevExtreme/blob/ed68457b6afcd0521d494707dd9bd3f0e5a7cbfe/js/ui/drop_down_editor/ui.drop_down_list.js#L22)

Reply:
fixed